universal-login-relayer/test/relayer/services/MessageHandler.js

```diff
@@ -71,20 +73,22 @@ describe('Relayer - MessageHandler', async () => {
       const message =  {...addKeyMessage, from: walletContract.address, gasToken: mockToken.address, to: walletContract.address};
       const signedMessage = await createSignedMessage(message, wallet.privateKey);
 
-      await messageHandler.executeSigned(signedMessage);
+      await messageHandler.handleMessage(signedMessage);
     });
 
     it('should remove key', async () => {
       expect((await walletContract.getKeyPurpose(otherWallet.address))).to.eq(ACTION_KEY);
       const message =  {...removeKeyMessage, from: walletContract.address, gasToken: mockToken.address, to: walletContract.address};
       const signedMessage = await createSignedMessage(message, wallet.privateKey);
 
-      await messageHandler.executeSigned(signedMessage);
+      await messageHandler.handleMessage(signedMessage);
       expect((await walletContract.keyExist(otherWallet.address))).to.eq(false);
     });
   });
 
   after(async () => {
+    await knex.delete().from('signature_key_pairs');
+    await knex.delete().from('messages');
```

Comment:
Here you can reuse clean database function extracted earlier.

Reply:
✔️